sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,15 +1,32 @@
-## Release History
+# Release History
 
-### 0.1.0b4 (unreleased)
+## 2.6.0 (2022-07-06)
```

Comment:
Why is this version 2.6.0? I don't think we'll be reaching that version number for years...

Reply:
This should be 0.1.0b5 for the July release.

Reply:
Minor confusion with the version number for the Azure ML CLI release. I'll fix this in another commit.

Reply:
Should be all good now.

Reply:
Thanks!